documentation/sections/announcing_coffeescript_2.md

```diff
@@ -0,0 +1,82 @@
+
+## Other Features
+
+Besides supporting new ES2015+ features and outputting older CoffeeScript features in ES2015+ syntax, CoffeeScript 2 has added support for the following:
+
+- [JSX](http://coffeescript.org/v2/#jsx)
+- [Line comments](http://coffeescript.org/v2/#comments) are now output (in CoffeeScript 1 they were discarded)
+- Block comments are now allowed anywhere, enabling [static type annotations](http://coffeescript.org/v2/#type-annotations) using Flow’s comment-based syntax
+
+There are many smaller improvements as well, such as to the `coffee` command-line tool. You can read all the details in the [changelog](http://coffeescript.org/v2/#changelog) for the 2.0.0 betas.
+
+## What About…
+
+A few features get asked about so often that we added a section to the docs called [Unsupported ECMAScript Features](http://coffeescript.org/v2/#unsupported). These include `let` and `const`, named functions and the `get` and `set` keywords. You can read the docs for the details, but simply put: `let` and `const` and named functions aren’t necessary for compatibility or interoperability with other libraries, and supporting them would add unwanted complexity to CoffeeScript; and getters and setters are supported but with the more verbose syntax.
+
+## Future Compatibility
+
+Back when CoffeeScript 1 was created, [Babel](babeljs.io) and ES2015 were both several years away. The CoffeeScript compiler itself had to do what today’s Babel compiler does, converting modern features like destructuring and arrow functions into equivalent ES3 JavaScript.
+
+But Babel exists now, and it does its one job—converting today’s and tomorrow’s JavaScript into yesterday’s—exceedingly well. With Babel around, there’s no need for the CoffeeScript compiler to duplicate its functionality. All the CoffeeScript compiler needs to worry about now is converting the CoffeeScript version of new syntax into the ES version of that syntax, e.g. `import fs from 'fs'` into `import fs from 'fs'`. The CoffeeScript compiler need not do all the work that Babel does. This makes adding support for new ES features much easier than before.
+
+Fortunately, most features added by ECMA in recent years haven’t required any updates at all in CoffeeScript. New global objects, or methods on global objects, are just supported and output as is. Want to use `Object.assign` in CoffeeScript? Just type `Object.assign`. Ditto for `Array.forEach` and `Array.map` and `String.includes` and any number of excellent additions to the language that ECMA has added. You may need to add [polyfills](https://babeljs.io/docs/usage/polyfill/) for these methods if your target runtime(s) don’t support them, but that would be no different than if you used these methods in a plain JavaScript project.
```

Comment:
This section about polyfills is also a candidate for shortening or removal if you want to make the whole thing shorter.